iaso/api/completeness_stats.py

```diff
@@ -332,60 +333,97 @@ def list(self, request: Request, *args, **kwargs) -> Response:
                 )
 
         def to_dict(row_ou: OrgUnitWithFormStat):
+            # TODO: improve this (check validation, account ,...)
+            children_count = OrgUnit.objects.all().filter(parent_id=row_ou.id).count()
```

Comment:
@mathvdh can you have a look on this please, we need to annotate the query and make it more robust

Reply:
@SimonDus